src/webextension/telemetry.js

```diff
@@ -7,3 +7,9 @@ export async function recordEvent(method, object, extra) {
     type: "proxy_telemetry_event", method, object, extra,
   });
 }
+
+export async function setScalar(name, value) {
```

Comment:
As with the handler in `message-ports.js`, this method is not used anywhere.

I think you included this out of completeness, which is ok if we see other front-end driven scalars coming in; in which case there should be a test.  Otherwise, it's fine to just remove this (and the `"proxy_telemetry_scalar"` handler) for now.